### PR TITLE
Pass through option parameters

### PR DESCRIPTION
**Pull request by Lajnold**
This allows setting 'disabled' per option, for example.

### CODE REVIEW

File: src/components/Select2.js [hunk 1]
```diff
@@ -60,8 +60,10 @@ export default class Select2 extends Component {
           if (typeof item === 'string' ||
             ((!!item && typeof item === 'object') && Object.prototype.toString.call(item) === '[object String]')) {
             return (<option key={'option-' + k} value={item}>{item}</option>);
+          } else {
```
Comment (rkit) [inline src/components/Select2.js]: Please, fix this: http://jmp.sh/L4JwgeQ ([rule](http://eslint.org/docs/rules/no-else-return.html))
The repository has a config for [eslint](https://github.com/rkit/react-select2-wrapper/blob/master/.eslintrc)

**Comment by rkit** [comment]
Thanks!

**Comment by rkit** [comment]
I released a new version (0.3.0)

**Comment by Lajnold** [comment]
Thanks!

**Pull request merged**